mkdocs/pages/en/devbook/transactions/complete-aggregate.md

```diff
@@ -0,0 +1,263 @@
+---
+title: Complete Aggregate
+---
+
+# Creating a Complete Aggregate Transaction
+
+This tutorial shows how to create an asset swap using <complete aggregate transactions:>.
+
+In this example, Account A sends 10 <XYM:> to Account B, while Account B simultaneously sends 1 custom <mosaic:> back
+to Account A:
+
+```dot
+digraph {
+    rankdir="LR";
+    fontsize=12;
+    subgraph clusterAggregate {
+        label = "Complete Aggregate Transaction";
+        tooltip = "Complete Aggregate Transaction";
+        subgraph clusterT1 {
+            label = "Embedded Transfer 1";
+            tooltip = "Embedded Transfer 1";
+            style = dashed;
+            A2 [label="Account A" tooltip="Account A"];
+            B2 [label="Account B" tooltip="Account B"];
+            A2 -> B2 [label="10 XYM"];
+        }
+        subgraph clusterT2 {
+            label = "Embedded Transfer 2";
+            tooltip = "Embedded Transfer 2";
+            style = dashed;
+            A1 [label="Account A" tooltip="Account A"];
+            B1 [label="Account B" tooltip="Account B"];
+            A1 -> B1 [label="1 Custom Mosaic" dir=back];
+        }
+    }
+}
+```
+
+Both parties coordinate off-chain to collect signatures before announcement, ensuring the swap executes as a
+single atomic transaction.
+
+!!! note "Two types of aggregate transactions"
+
+    <Aggregate transactions:> group multiple <transactions:> in single operation, and require <signatures:> from all
+    involved accounts.
+
+    A _complete aggregate transaction_ collects all signatures before being announced.
+    This works well in two scenarios:
+
+    * **Multi-party coordination:** When parties can communicate off-chain to exchange transaction payloads and
+      cosignatures.
```

Comment:
This is now the first usage of the term "cosignature", so the link to the definition should be moved here.

Also, I realize the definition of cosignature is a bit ambiguous: it should clarify that it applies to multiple signatures from different accounts. I'll fix that in the Textbook.